xds/internal/balancer/edsbalancer/eds_impl.go

```diff
@@ -389,6 +391,18 @@ func (edsImpl *edsBalancerImpl) handleSubConnStateChange(sc balancer.SubConn, s
 	}
 }
 
+// updateConfig handles changes to the circuit breaking configuration.
+func (edsImpl *edsBalancerImpl) updateConfig(edsConfig *EDSConfig) {
+	if edsImpl.counter == nil || edsImpl.counter.ServiceName != edsConfig.EDSServiceName {
```

Comment:
Let's protect this with an env variable `GRPC_XDS_EXPERIMENTAL_CIRCUIT_BREAKING`. So if this is set to false, `edsImpl.counter` is never set (is always nil).

Add the env variable like this: https://github.com/grpc/grpc-go/blob/master/xds/internal/env/env.go#L41

Reply:
Created the environment variable.